src/IIIFPresentation/API.Tests/Helpers/CollectionHelperXTests.cs

```diff
@@ -90,7 +108,13 @@ public void GenerateFlatCollectionViewId_CreatesViewId()
         var collection = new Collection()
         {
             Id = "test",
-            Slug = "slug"
+            Hierarchy =
+            [
+                new Hierarchy()
+                {
+                    Slug = "slug"
```

Comment:
Would it be worth turning "slug" into a const somewhere, like `defaultSlugName`?